packages/terra-props-table/src/PropsTable.jsx

```diff
@@ -66,17 +70,19 @@ const PropsTable = ({ componentName, src, ...customProps }) => {
    */
   const componentProps = componentMetaData.props;
 
+  const tableEntryClass = cx('prop-table-element');
+
   return (
     <div dir="ltr" className="markdown-body">
       <h2>{componentName} Props</h2>
       <table {...customProps}>
         <thead>
           <tr>
```

Comment:
You should be able to add the `prop-table-elemenet` class to this table row, instead of each table cell to achieve the vertical-top alignment